extensions/azdata/src/typings/azdata-ext.d.ts

```diff
@@ -196,6 +196,32 @@ declare module 'azdata-ext' {
 							memory: string // "256Mi"
 						}
 					}
+				},
+				roles: {
```

Comment:
@bergeron this is how I believe the spec to be updated, could you confirm please?

Reply:
I think the role should be "worker" singular.  Can you test it against a recent cli+backend build to be sure?

---
In reply to: [612667940](https://github.com/microsoft/azuredatastudio/pull/15114#discussion_r612667940) [](ancestors = 612667940)